src/components/MAPI/clusters/CreateCluster/CreateClusterControlPlaneNodeAZs.tsx

```diff
@@ -0,0 +1,133 @@
+import AZSelection from 'Cluster/AZSelection/AZSelection';
+import {
+  AvailabilityZoneSelection,
+  AZSelectionVariants,
+  AZSelectionZonesUpdater,
+  IUpdateZoneLabelsPayload,
+} from 'Cluster/AZSelection/AZSelectionUtils';
+import { Cluster, ControlPlaneNode } from 'MAPI/types';
+import { determineRandomAZs, getSupportedAvailabilityZones } from 'MAPI/utils';
+import PropTypes from 'prop-types';
+import React, { useEffect, useMemo, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { RUMActions } from 'shared/constants/realUserMonitoring';
+import { getProvider } from 'stores/main/selectors';
+import InputGroup from 'UI/Inputs/InputGroup';
+
+import { computeControlPlaneNodesStats } from '../ClusterDetail/utils';
+import {
+  IClusterPropertyProps,
+  withClusterControlPlaneNodeAZs,
+} from './patches';
+
+interface ICreateClusterControlPlaneNodeAZsProps
+  extends IClusterPropertyProps,
+    Omit<
+      React.ComponentPropsWithoutRef<typeof InputGroup>,
+      'onChange' | 'id'
+    > {}
+
+const CreateClusterControlPlaneNodeAZs: React.FC<ICreateClusterControlPlaneNodeAZsProps> = ({
```

Comment:
Nothing too special here, it's mostly the same as the `WorkerNodesCreateNodePoolAvailabilityZones` component